integration/qbft/scenarios/regular_attester.go

```diff
@@ -0,0 +1,181 @@
+package scenarios
+
+import (
+	"fmt"
+
+	specqbft "github.com/bloxapp/ssv-spec/qbft"
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	spectestingutils "github.com/bloxapp/ssv-spec/types/testingutils"
+
+	protocolstorage "github.com/bloxapp/ssv/protocol/v2/qbft/storage"
+)
+
+// RegularAttester integration test.
+// TODO: consider accepting scenario context - initialize if not passed - for scenario with multiple nodes on same network
+func RegularAttester() *IntegrationTest {
+	identifier := spectypes.NewMsgID(spectestingutils.Testing4SharesSet().ValidatorPK.Serialize(), spectypes.BNRoleAttester)
+
+	return &IntegrationTest{
+		Name:             "regular attester",
+		OperatorIDs:      []spectypes.OperatorID{1, 2, 3, 4},
+		Identifier:       identifier,
+		InitialInstances: nil,
+		Duties: map[spectypes.OperatorID][]scheduledDuty{
+			1: {scheduledDuty{Duty: createDuty(spectestingutils.Testing4SharesSet().ValidatorPK.Serialize(), spectestingutils.TestingDutySlot, 1, spectypes.BNRoleAttester)}},
+			2: {scheduledDuty{Duty: createDuty(spectestingutils.Testing4SharesSet().ValidatorPK.Serialize(), spectestingutils.TestingDutySlot, 1, spectypes.BNRoleAttester)}},
+			3: {scheduledDuty{Duty: createDuty(spectestingutils.Testing4SharesSet().ValidatorPK.Serialize(), spectestingutils.TestingDutySlot, 1, spectypes.BNRoleAttester)}},
+			4: {scheduledDuty{Duty: createDuty(spectestingutils.Testing4SharesSet().ValidatorPK.Serialize(), spectestingutils.TestingDutySlot, 1, spectypes.BNRoleAttester)}},
+		},
+		InstanceValidators: map[spectypes.OperatorID][]func(*protocolstorage.StoredInstance) error{
+			1: {
+				regularAttesterInstanceValidator(1, identifier),
+			},
+			2: {
+				regularAttesterInstanceValidator(2, identifier),
+			},
+			3: {
+				regularAttesterInstanceValidator(3, identifier),
+			},
+			4: {
+				regularAttesterInstanceValidator(4, identifier),
+			},
+		},
+		StartDutyErrors: map[spectypes.OperatorID]error{
+			1: nil,
+			2: nil,
+			3: nil,
+			4: nil,
+		},
+	}
+}
+
+func regularAttesterInstanceValidator(operatorID spectypes.OperatorID, identifier spectypes.MessageID) func(actual *protocolstorage.StoredInstance) error {
+	return func(actual *protocolstorage.StoredInstance) error {
+		encodedConsensusData, err := spectestingutils.TestAttesterConsensusData.Encode()
+		if err != nil {
+			return fmt.Errorf("encode consensus data: %w", err)
+		}
+
+		proposalData, err := (&specqbft.ProposalData{
+			Data:                     encodedConsensusData,
+			RoundChangeJustification: nil,
+			PrepareJustification:     nil,
+		}).Encode()
+		if err != nil {
+			return fmt.Errorf("encode proposal data: %w", err)
+		}
+
+		prepareData, err := (&specqbft.PrepareData{
+			Data: encodedConsensusData,
+		}).Encode()
+		if err != nil {
+			panic(err)
```

Comment:
https://github.com/bloxapp/ssv/pull/793#discussion_r1067178852

Reply:
it overwrites on future brunches